js/compiler.ts

```diff
@@ -400,7 +401,8 @@ export class DenoCompiler implements ts.LanguageServiceHost {
    */
   getModuleDependencies(
     moduleSpecifier: ModuleSpecifier,
-    containingFile: ContainingFile
+    containingFile: ContainingFile,
+    byModuleName = false
```

Comment:
I don't think we need a non-by module name variant of this...  The only reason it was ever here was to support the module names, which we didn't have exposed... I would just always assume it will be by module name.